sdcm/argus_test_run.py

```diff
@@ -211,26 +244,29 @@ def warn_once(cls, message: str, *args: list):
         LOGGER.warning(message, *args)
 
     @classmethod
-    def from_sct_config(cls, test_id: UUID, test_module_path: str,
-                        sct_config: SCTConfiguration) -> TestRunWithHeartbeat:
+    def from_sct_config(cls, test_id: UUID, sct_config: SCTConfiguration) -> TestRunWithHeartbeat:
         # pylint: disable=too-many-locals
         if cls.TESTRUN_INSTANCE:
             raise ArgusTestRunError("Instance already initialized")
 
-        release_name = os.getenv("GIT_BRANCH", get_git_current_branch()).split("/")[-1]
-        if release_name not in cls.AVAILABLE_RELEASES:
-            raise ArgusTestRunError("Refusing to track a non-whitelisted branch", release_name, cls.AVAILABLE_RELEASES)
         LOGGER.info("Preparing Test Details...")
-        test_group, *_ = test_module_path.split(".")
-        config_files = sct_config.get("config_files")
         job_name = get_job_name()
+        if job_name == "local_run":
+            raise ArgusTestRunError("Will not track a locally run job")
+
         job_url = get_job_url()
+        release_name, group_name, test_name = parse_run_details(job_name.split("/"))
+        if not release_name:
+            raise ArgusTestRunError(f"Unable to track a job: {job_name}", job_name)
```

Comment:
Is it intended to pass `job_name` as a second argument?

Reply:
yes